hexrd/ipfcolor/colorspace.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+# Written by Saransh Singh <saransh1@llnl.gov> and others.
+# LLNL-CODE-529294.
+# All rights reserved.
+#
+# This file is part of HEXRD. For details on dowloading the source,
+# see the file COPYING.
+#
+# Please also see the file LICENSE.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License (as published by the Free
+# Software Foundation) version 2.1 dated February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE. See the terms and conditions of the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program (see file LICENSE); if not, write to
+# the Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+# Boston, MA 02111-1307 USA or visit <http://www.gnu.org/licenses/>.
+# =============================================================================
```

Comment:
FYI, I think hexrd is under a 3-clause BSD license now, instead of LGPL 2.1, although I do see that this header is in some of the other files too...

Is this right, @cjh1 ?

Reply:
That is correct: https://github.com/HEXRD/hexrd/blob/master/LICENSE

Reply:
Actually, yes it is.  I have been meaning to implement some sort of 'headerizing' function, because LLNL wants all the files to have a header.  It does need updating.